package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@wearefrank/ladybug",
-  "version": "0.0.60",
+  "version": "00.00.00",
```

Comment:
Hoort deze versie op 00.00.00 te staan?

Reply:
@philipsens, ik heb dit zo gedaan om te laten zien dat dit versienummer niet relevant is. Je kan in een JSON file geen comment zetten, dus ik weet geen betere manier om dit te tonen. Het versienummer in pom.xml is bepalend voor de versie die gereleased wordt.

Reply:
En als er helemaal geen versienummer staat, dan weet ik niet of de package.json nog geldig zou zijn voor yarn.

Reply:
I am speaking to @jacodg right now. He points out that the version number of the frontend is shown in the UI. Probably version number 00.00.00 will appear if we do not take care of this. I will put this PR to draft and test this.

Reply:
We can postpone this. I make this PR ready-for-review again.